### DISCUSSION TITLE
RGB Automatization Asus Rog Strix G16 2023

### DISCUSSION THREAD

**Discussion by erickplc16** [seerge/g-helper#824] [Q&A]
Just wanted to know if its possible to turn RGB Lights when connected and then turn them off when on battery mode automatically?

if possible how can I do it?

I don't see an available option:
![image](https://github.com/seerge/g-helper/assets/139263334/ac9e5519-1d8c-4024-8507-8e5e2d5fe08a)

**Comment by seerge** [comment]
@erickplc16 app remembers backlight brightness individually per battery / plugged. But you need to stop asus services first, otherwise Asus Optimization service that controls backlight will intefrere.

**Reply by erickplc16** [reply]
I just used the uninstall tool because of the remaining services, restart but it didnt work:

![image](https://github.com/seerge/g-helper/assets/139263334/d21020a9-5637-4097-a03c-4209ceb73018)

**Reply by seerge** [reply]
@erickplc16 just set some backlight brightness (with regular hotkeys) on battery and then plug and set another one, and then unplug :) it should adjust it 

If it's not happening upload a log and config from ``%appdata%\ghelper`` here

**Reply by erickplc16** [reply]
@seerge thanks, it worked!
